objc-xcode-warnings-errors/FISAppDelegate.m

```diff
@@ -6,7 +6,44 @@ @implementation FISAppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     // Override point for customization after application launch.
-    return YES;
+    
+    NSString *unused = @"This variable generates a warning because it is unused.";
+    
+//    NSLog(@"%@", unused);
+    
+//    NSInteger *i = 0;
+//    NSLog(@"i: %li", i);
+    
+//    NSInteger x = i + 1;
+//    NSLog(@"x: %li", x);
+    
+    NSLog(@"Anything after the return statement will not get executed.");
+    
+    return YES; // this line ends the method
+    
+    NSLog(@"Take note that this line doesn't print to the console.");
 }
 
+//NSString *message = @"Even though they don't belong here, the compiler won't actually complain about string literals or primitives defined outside of a method body (which is held by  curly braces {...} ), but...";
+
+//NSInteger j = 0;
+//BOOL itIsKnownKhaleesi = YES;
+
+//NSLog(@"...any statements containing function calls, operations, or method calls will produce errors.");
+
+
+
+
+//NSLog(@"%@", message);
+
+
+
+//j++;
+
+//itIsKnownKhaleesi = NO;
+
+
+//NSString *useless = [NSString stringWithString:@"Which means the variables above, while permitted, are completely useless."];
```

Comment:
(Not strictly true, since this is how you do constants. But they're definitely not what the student intends to make at this point in the course.)